client/public/_redirects

```diff
@@ -0,0 +1 @@
+/* /index.html 200
```

Comment:
We need others in here also like
/Dashboard  /dashboard
/Billing  /billing
etc..

Reply:
From what I just tested I don't know if that changes anything but I added them anyway.